techniques/aria/ARIA4.html

```diff
@@ -51,57 +51,51 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#tests">Tests</a></li>
-<li><a href="#test-rules">Test Rules</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#tests">検証</a></li>
+<li><a href="#test-rules">検証ルール</a></li>
```

Comment:
今のところはほかで「テストルール」としているので、そのようにしてください。

```suggestion
<li><a href="#test-rules">テストルール</a></li>
```